check-peer-deps.js

```diff
@@ -1,14 +1,16 @@
 const { exec } = require('sb-exec');
 const semver = require('semver');
+const commandLineArgs = require('command-line-args');
+const commandLineUsage =  require('command-line-usage');
 
 // Flags
 // Enable debug output
-const DEBUG = false;
+let DEBUG = false;
```

Comment:
The default values should only be in one place, since it looks like `command-line-args` `OptionDefinition` has a `defaultValue` property let's define it there.

This and the following definitions should be left as `let DEBUG`.